tests/conftest.py

```diff
@@ -1,11 +1,19 @@
 """
 Custom test collector for our integration tests.
 
-Each directory that has a script named 'verify' is considered
-a test. jupyter-repo2docker is run on that directory,
-and then ./verify is run inside the built container. It should
-return a non-zero exit code for the test to be considered a
-success.
+
+Test lifecycle:
+- Find all directories that contain `verify` or `*.repos.yaml`
+- If `verify` is found:
+    - Run `jupyter-repo2docker` on the test directory.
+      - Extra arguments may be added as YAML list of strings in `extra-args.yaml`.
```

Comment:
took me a while to find this, hence updating this section

Reply:
Nice clarification!